.changeset/little-pillows-fry.md

```diff
@@ -0,0 +1,15 @@
+---
+"@salt-ds/core": minor
+---
+
+`Text` component has two new variants: `TextAction` and `TextNotation`. See [Storybook](https://storybook.saltdesignsystem.com/?path=/story/core-text-text-qa--all-variants-grid) for more information.
+
+```tsx
+<Text styleAs="notation" as="span">Lorem ipsum</Text>
+<Text styleAs="action" as="span">Lorem ipsum</Text>
+```
+
+```tsx
+<TextNotation>Lorem ipsum</TextNotation>
```

Comment:
Do we want TextNotation, TextAction

Reply:
@joshwooding do you mean we might not want these components altogether  `TextNotation / TextAction`, and have them available only through `styleAs` prop, and shown in examples on site?

Reply:
Exactly yeah, Notation is one I'm unsure about but I don't really see people using TextAction often?

Reply:
Hmm I understand where you're coming from - I'm just conscious if we derail from the "format" of offering all available `styleAs` variants in individual components form, could it be seen as a mistake/error on our end that maybe our documentations just aren't up to date etc. If we do remove the components TextNotation/TextAction we'll have document that well so users won't assume at first glance components they can import are the only ones available for use, and to use action or notation the only way is through `styleAs`

Reply:
I don't know if we've ever had the stance that we'll always have equivalence. It's a fair enough assumption, though but it creates redundancy. We can keep TextNotation/TextAction and always reassess this decision later.